### ISSUE TITLE
scoring target DLET for plan 2 strangely low

### ISSUE THREAD

**Issue by nbassler** [APTG/2022_DCPT_LET_timepix#8]
and varies wildly to TLET
also primary number is not constant

plan 4a and plan4b also seems to have similar problems.

**Comment by nbassler** [comment]
@grzanka could be a bug in SH12A

**Comment by nbassler** [comment]
ran 1e6 primaries for plan 2. Preliminary findings: 
- only neutrons crossed the target region
- sh_score_point() caught a few events, where neutrons with bad PDG numbers were scored as charged particles for dose.
- for LET particles, uncharged particles are excluded, but should be investigated further. 

Will be investigated in https://gitlab.au.dk/shieldhit/shieldhit/-/issues/439

**Comment by mdsainth** [comment]
Particularly the results from LET in water are very low. The LET in target (e.g. NB_target_pos2_L__p06.txt) seem reasonable.

**Comment by mdsainth** [comment]
@nbassler did you increase the scoring volume to enhance stats?

**Comment by nbassler** [comment]
Increasing scoring volume not needed at the moment. :) I try to understand what is going on, so any event happening repoducible is useful.

What I found out so far is:

- at that distance, the detector is mostly hit by neutrons (surprise! :)
- the way neutrons are treated is almost philosophical discussion. SH12A under certain conditions makes secondary charged partciles, and sometimes not, and just dumps the neutron energy in a point (this can be filtered).
- primary particles never reached the detector
- unspecific energy deposition in a point by a neutron has no charged particles, therefore no LET, and also no dose-to-water (unless medium is water)
- I did find a bug in SH12A, when charged particles below transport threshold are created, then the particle was passed to scoring as parent particle. E.g. proton scattering elastically on carbon... carbon gets Ekin, carbon deposits energy Ekin... Ekin is scored deposited by proton... Oops. But this happens very rarely, like one in a million primaries in this case. 
- I did see very few secondary charged particles, made from neutrons. They may have been scored wrongly bc of the above mentioned issue. I fixed it, but need to test more, and then review from @grzanka and release. Will take a bit of more time. :)

Generally, I think we are at the border of what is possible with SH12A. MCNP* may be better choice here.
